### PR TITLE
Deprecate cvss fields in APIs and use FlawCVSS model for validations

### PR DESCRIPTION
**Pull request by jobselko**
This PR updates the following:
- `cvss2`, `cvss2_score`, `cvss3`, `cvss3_score`, `nvd_cvss2`, and `nvd_cvss3` fields are now deprecated in Flaw API
- `cvss2`, `cvss2_score`, `cvss3`, and `cvss3_score` fields are now deprecated in Affect API
- flaw validations now use the new `FlawCVSS` model instead of `cvss3_score`, `nvd_cvss3`, and `cvss3` fields from the `Flaw` model

Closes OSIDB-1105

### CODE REVIEW

File: osidb/models.py [hunk 1]
```diff
@@ -846,48 +846,67 @@ def __str__(self):
         """convert to string"""
         return str(self.uuid)
 
-    def _validate_rh_nvd_cvss_score_diff(self):
+    def _validate_rh_nist_cvss_score_diff(self):
         """
-        Checks that the difference between the RH and NVD CVSS score is not >= 1.0
+        Checks that the difference between the RH and NIST CVSSv3 score is not >= 1.0.
```
Comment (jobselko) [inline osidb/models.py:851]: I changed `NVD` to `NIST` everywhere to keep the name consistent.

**Comment by jobselko** [comment]
I intentionally did not deprecate cvss fields in `models.py`, so there is no extra work in `filters.py` now. Deprecation/removal can be done after a major release, when those fields are no longer needed.

File: osidb/api_views.py [hunk 2]
```diff
@@ -345,6 +345,43 @@ def include_exclude_fields_extend_schema_view(
...
+                location=OpenApiParameter.QUERY,
+                deprecated=True,
+            ),
+            OpenApiParameter(
+                "nvd_cvss3",
+                type=OpenApiTypes.STR,
+                location=OpenApiParameter.QUERY,
+                deprecated=True,
+            ),
+            # todo: cvss3 comment z meta attr?
```
Comment (osoukup) [inline osidb/api_views.py:384]: I think that there is no need to reflect the meta attribute changes in the API docs. There are no guarantees regarding the metadata.

**Review by osoukup — APPROVED** [review]
LGTM

File: osidb/api_views.py (hunk 2 above)
Comment (jobselko) [reply osidb/api_views.py:384]: This does not belong here. It is just my old note.

**Pull request merged**